resources/test/fixtures/flake8_pytest_style/PT003.py

```diff
@@ -14,3 +14,16 @@ def ok_other_scope():
 @pytest.fixture(scope="function")
 def error():
     ...
+
+@pytest.fixture(scope="function", name="my_fixture")
+def error_multiple_args():
+    ...
+
+
+@pytest.fixture(name="my_fixture", scope="function", **kwargs)
+def error_second_arg():
+    ...
+
+@pytest.fixture("my_fixture", scope="function", **kwargs)
+def error_arg():
+    ...
```

Comment:
I think there are still a few cases that aren't quite handled correctly:

```py
@pytest.fixture(
    scope="function",
    name="my_fixture",
)
def error_multiple_args():
    ...


@pytest.fixture(name="my_fixture", scope="function")
def error_multiple_args():
    ...
```

These give:

```py
@pytest.fixture(
    
    name="my_fixture",
)
def error_multiple_args():
    ...


@pytest.fixture(name="my_fixture", )
def error_multiple_args():
    ...
```

Sorry for all the churn, but do you mind tweaking to address those?

Reply:
Will do, should have tested a bit better

Reply:
Na don't sweat it. I only know to try these because I've run into them before :joy:

Reply:
Would be nice to turn them into utilities after (e.g. `remove_argument`)

Reply:
Strongly agree!